### PR TITLE
st2 [2431][UPD] Update dotfiles

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[2431](https://www.quartile.co/web?debug=1#id=2431&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: tests
+
+on:
+  pull_request:
+    branches:
+      - "12.0*"
```
Comment (yostashiro) [inline .github/workflows/test.yml:6]:
```suggestion
      - "st*"
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
@AungKoKoLin1997 Please update the content so that the pre-commit and CI run on pulls and pushes to st* branches.

File: .github/workflows/pre-commit.yml [hunk 2]
```diff
@@ -2,7 +2,12 @@ name: pre-commit
 
 on:
   pull_request:
+    branches:
+      - "st*"
   push:
+    branches:
+      - "12.0"
+      - "12.0-ocabot-*"
```
Comment (yostashiro) [inline .github/workflows/pre-commit.yml]:
```suggestion
      - "st*"
```

@AungKoKoLin1997 What is your understanding on these records (why no changes were made)?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+name: tests
+
+on:
+  pull_request:
+    branches:
+      - "st*"
+  push:
+    branches:
+      - "12.0"
+      - "12.0-ocabot-*"
```
Comment (yostashiro) [inline .github/workflows/test.yml]:
```suggestion
      - "st*"
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    name: ${{ matrix.name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo12.0:latest
+            makepot: "true"
+            name: test with Odoo
```
Comment (yostashiro) [inline .github/workflows/test.yml]:
```suggestion
```
We want to skip the test on odoo/odoo as we are not using it.

**Review by yostashiro — CHANGES_REQUESTED** [review]
@AungKoKoLin1997 Can you please explain how .github/workflow/stale.yml works?

**Comment by AungKoKoLin1997** [comment]
> @AungKoKoLin1997 Can you please explain how .github/workflow/stale.yml works?

After a period of inactivity , a label will be applied to mark an issue or pull request as stale.If the Issue or Pull Request is updated, or anyone comments, then the stale label is removed. If no more activity occurs, the Issue or Pull Request will be automatically closed.

File: .github/workflows/pre-commit.yml (hunk 2 above)
Comment (AungKoKoLin1997) [reply .github/workflows/pre-commit.yml]: > @AungKoKoLin1997 What is your understanding on these records (why no changes were made)?

Sorry. I misunderstood about it. I confused about the push. Now I got it.

**Comment by yostashiro** [comment]
> > @AungKoKoLin1997 Can you please explain how .github/workflow/stale.yml works?
> 
> After a period of inactivity , a label will be applied to mark an issue or pull request as stale.If the Issue or Pull Request is updated, or anyone comments, then the stale label is removed. If no more activity occurs, the Issue or Pull Request will be automatically closed.

I get the general idea but wasn't sure if it would work in an non-OCA repo. Do you have an insight on this?

**Review by yostashiro — APPROVED** [review]
Thanks for the fix.  Let's proceed with this.

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Can you please follow up with another PR to fix the pre-commit and CI issues?

**Pull request merged**